_plugins/hooks/sitemap.rb

```diff
@@ -0,0 +1,9 @@
+require 'pry'
+
+Jekyll::Hooks.register :documents, :pre_render do |doc|
+
+  if doc.collection.label == 'pages'
+    doc.data['sitemap'] = !doc.data['search_excluded']
```

Comment:
is `search_excluded` always going to true/false? My understanding is they are trinary (true, false, null). Is that going to bork anything?

Reply:
@j-brammer I guess I'm really just wondering what happens when that radio isn't checked.

Reply:
☝️This a good catch @lukemoderwell. 

I would assume that anything not checked will return undefined, which would evaluate to true in this expression.

Reply:
@tcmacdonald Jason just tested and you are correct. Undef will return `false` which, since we are taking the opposite, gives us the desired behavior. No radio selection = include in sitemap.